backend/apps/response/notification_dispatcher.py

```
@@ -116,6 +117,16 @@ def dispatch(self, report_data: Dict[str, Any], mr_info: Dict[str, Any], project
 
             return summary
 
+        except ImproperlyConfigured as config_error:
+            elapsed_time = time.time() - start_time
```

Ruff (0.14.6)

Undefined name `start_time`

(F821)